src/daos/docker/s3-tests/setup.sh

```diff
@@ -3,16 +3,16 @@
 # create conf file from s3tests.conf.SAMPLE
 # prepare the tests
 
-# source $CEPH_PATH/src/daos/silent_pushd_popd.sh
+source $CEPH_PATH/src/daos/silent_pushd_popd.sh
 
-if [[ ! "$ceph_folder" =~ . ]];
+if [[ ! "$CEPH_PATH" =~ . ]];
 then
-    ceph_folder=/opt/ceph
+    CEPH_PATH=/opt/ceph
 fi
```

Comment:
We're changing the value of CEPH_PATH after importing a source, is this intended?

Reply:
ugh, good catch.  I'll fix it.  I meant to delete that